doc-content/enterprise-only/project-data/test-designer-alias-con.adoc

```diff
@@ -0,0 +1,4 @@
+[id='test-designer-alias-con']
+= Concept of aliases (for rule-based scenarios only)
+
+In the Test Scenario designer, once you map a header cell with a data object, the data object is removed from the *Test Editor* tab in the *Test Tools* panel. You can re-map a data object to another header cell by using the concept of *Alias*. Aliases enable you to specify multiple instances of the same data object. Ensure that the aliases are uniquely named. By doing so, the data object again becomes available in the *Test Editor* tab for remapping. To introduce an alias, double click on a corresponding header cell and manually change the name. You can create property aliases as well. This allows you to rename the used properties directly in the table which enhances user experience.
```

Comment:
Minor suggestion - remove "which enhances user experience.":

s/This allows you to rename the used properties directly in the table which enhances user experience./This allows you to rename the used properties directly in the table.

Reply:
Changed as suggested.